### ISSUE TITLE
Pretrained model results different from that in README.md

### ISSUE THREAD

**Issue by tomato18463** [ifzhang/ByteTrack#365]
Hi,

My steps were as follows:

- Datasets were prepared as instructed in README.md
- Download MOT17 bytetrack_s_mot17 model as given in README.md
- Run `tools/track.py` as in README.md. Line 19 and Line 101 in exps/example/mot/yolox_s_mix_det.py were changed to output results for training and test set
- Run ` tools/interpolation.py` as in README.md.
- Since the above codes only produced results for *-FRCNN.txt for the training set, I copied the produced *-FRCNN.txt file to get the corresponding *-DPM.txt and *-SDP.txt files for the training set.
- The above steps gave me 42 .txt files, and I submitted them to MOT 17 evaluation server.
- I got a MOTA of 75.45, while the number in README.md is 79.2

Was there anything wrong with my steps? Why did I get a lower score than the one in README.md?

Thanks for your help.

**Comment by Long00River** [comment]
Hey ! Did you sort out that issue? I tried the frame interpolation post-processing from the paper and pushed accuracy to 79+, but can't crack 80.3. Any clue how to tune the threshold?
